services/ordering/src/EShopOnAbp.OrderingService.Application.Contracts/Permissions/OrderingServicePermissions.cs

```diff
@@ -2,10 +2,17 @@
 
 namespace EShopOnAbp.OrderingService.Permissions
 {
-    public class OrderingServicePermissions
+    public static class OrderingServicePermissions
     {
         public const string GroupName = "OrderingService";
 
+        public static class Orders
+        {
```

Comment:
Other methods used by administration should have permissions. (ex `GetOrdersAsync`)

Reply:
`GetOrderAsync` already uses permission.